CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+├── bundle.Dockerfile
+├── config
+│   ├── certmanager
+│   │   ├── certificate.yaml
+│   │   ├── kustomization.yaml
+│   │   └── kustomizeconfig.yaml
+│   ├── crd
+│   │   ├── bases
+│   │   │   ├── aiml.pachyderm.com_pachydermexports.yaml
+│   │   │   └── aiml.pachyderm.com_pachyderms.yaml
+│   │   ├── kustomization.yaml
+│   │   ├── kustomizeconfig.yaml
+│   │   └── patches
+│   │       ├── cainjection_in_pachydermexports.yaml
+│   │       ├── cainjection_in_pachyderms.yaml
+│   │       ├── webhook_in_pachydermexports.yaml
+│   │       └── webhook_in_pachyderms.yaml
+│   ├── default
+│   │   ├── kustomization.yaml
+│   │   ├── manager_auth_proxy_patch.yaml
+│   │   ├── manager_config_patch.yaml
+│   │   ├── manager_webhook_patch.yaml
+│   │   └── webhookcainjection_patch.yaml
+│   ├── manager
+│   │   ├── controller_manager_config.yaml
+│   │   ├── kustomization.yaml
+│   │   └── manager.yaml
+│   ├── manifests
+│   │   ├── bases
+│   │   │   └── pachyderm-operator.clusterserviceversion.yaml
+│   │   └── kustomization.yaml
+│   ├── prometheus
+│   │   ├── kustomization.yaml
+│   │   └── monitor.yaml
+│   ├── rbac
+│   │   ├── auth_proxy_client_clusterrole.yaml
+│   │   ├── auth_proxy_role_binding.yaml
+│   │   ├── auth_proxy_role.yaml
+│   │   ├── auth_proxy_service.yaml
+│   │   ├── kustomization.yaml
+│   │   ├── leader_election_role_binding.yaml
+│   │   ├── leader_election_role.yaml
+│   │   ├── pachyderm_editor_role.yaml
+│   │   ├── pachydermexport_editor_role.yaml
+│   │   ├── pachydermexport_viewer_role.yaml
+│   │   ├── pachyderm_viewer_role.yaml
+│   │   ├── role_binding.yaml
+│   │   ├── role.yaml
+│   │   └── service_account.yaml
+│   ├── samples
+│   │   ├── aiml_v1beta1_pachydermexport.yaml
+│   │   ├── aiml_v1beta1_pachyderm.yaml
+│   │   └── kustomization.yaml
+│   ├── scorecard
+│   │   ├── bases
+│   │   │   └── config.yaml
+│   │   ├── kustomization.yaml
+│   │   └── patches
+│   │       ├── basic.config.yaml
+│   │       └── olm.config.yaml
+│   └── webhook
+│       ├── kustomization.yaml
+│       ├── kustomizeconfig.yaml
+│       ├── manifests.yaml
+│       └── service.yaml
+├── CONTRIBUTING.md
+├── controllers
+│   ├── backup_job.go
+│   ├── diff.go
+│   ├── errors.go
+│   ├── generators
+│   │   ├── images.go
+│   │   ├── manifests.go
+│   │   └── template.go
+│   ├── initialize_database.go
+│   ├── pachyderm_controller.go
+│   ├── pachydermexport_controller.go
+│   └── suite_test.go
+├── Dockerfile
+├── go.mod
+├── go.sum
+├── hack
+│   ├── boilerplate.go.txt
+│   ├── bundle_prep.sh
+│   ├── catalogsource.yaml
+│   └── charts
+│       ├── 2.0.0
+│       │   ├── images.json
+│       │   ├── pachyderm-2.0.0-rc.1.tgz
+│       │   └── values.yaml
+│       └── 2.0.2
+│           ├── images.json
+│           ├── pachyderm-2.0.2.tgz
+│           └── values.yaml
+├── main.go
+└── PROJECT
+
+# Testing the Operator
+
+**TODO**: Fill this in. Open Questions:
+- How are operators installed in a running Kubernetes cluster?
```

Comment:
This question is very broad. I need clarification to be able to answer conclusively.